devops/templates/requirements-installation-steps-template.yml

```diff
@@ -0,0 +1,20 @@
+parameters:
```

Comment:
Could you add a comment giving an overview of what this template does? The condition syntax isn't the nicest, so a brief guide would be good.

Reply:
Yes, but I'll delay to a further PR. This wasn't created in this PR anyway, just moved here.